docs/visual-basic/programming-guide/concepts/linq/how-to-filter-on-an-attribute-xpath-linq-to-xml.md

```diff
@@ -8,7 +8,7 @@ This topic shows how to get the descendant elements with a specified name, and w
   
  The XPath expression is:  
   
- `.//Address[@Type='Shipping']`  
+ `./Address[@Type='Shipping']`  
```

Comment:
this is not a path change. Why is this included here?

Reply:
My bad... I was fixing a previous replace error with this. 
I'll correct this right now!